pkg/ibmvpc/analysis_output_test.go

```diff
@@ -480,6 +483,22 @@ var tests = []*vpcGeneralTest{
 		grouping:    true,
 		format:      vpcmodel.DRAWIO,
 	},
+	// resource group filtering example
+	// // ete-storage-project, ete-backup-and-storage expected to be filtered out
+	{
+		inputConfig:   "multi_resource_groups",
```

Comment:
Missing a test that validates filtering of tgw by resource group, where in the input config file there are multiple TGW objects, some with the input resource-group (that should be analyzed) and some without it (that should be filtered).
This test only covers filtering of tgw due to missing TGW configs.

Reply:
updated the test

Reply:
great, looks good. 
please also update the comment in the test file about the expected TGW objects filtered out by resource group.